dup.css

```
@@ -0,0 +1,240 @@
+    top: 0;
+    left: 50%;
+    width: 50%;
+    height: 100%;
+    overflow: hidden;
+    transition: transform 0.6s ease-in-out;
+    z-index: 100;
+}
+
+.overlay {
+    background: #4A90E2;
+    background: -webkit-linear-gradient(to right, #4A90E2, #50E3C2);
```

Biome (2.1.2)

[error] Unexpected nonstandard direction

You should fix the direction value to follow the syntax.
See MDN web docs for more details.

(lint/correctness/noInvalidDirectionInLinearGradient)